_i18n/en/resources/user-guides/monerod-as-a-systemd-service.md

```diff
@@ -0,0 +1,123 @@
+# Using monerod as a systemd Service
+
+If you are using a Linux distribution that uses `systemd`, you can use its
+service units to manage `monerod`.
+
+We will do this via the command line & assume that you have already built or
+downloaded `monerod`.
+
+
+## Create a User
+
+For security purposes, we should create a new user to run the monerod service:
+
+    useradd -r -m /var/lib/monero/ -s /usr/bin/nologin monero
+
+This will create a `monero` system account with no login shell and a home
+directory in `/var/lib/monero/`.
+
+
+## Create the Configuration File
+
+Create `/etc/monerod.conf` with the following content:
+
+    # Configuration for monerod
+    data-dir=/var/lib/monero
+    log-level=0
+
+If you are running a remote node, you may want to add the `restricted-rpc`,
+`rpc-bind-ip=X.X.X.X` & `confirm-external-bind` options as well.
+
+For a complete list of flags, run `monerod --help`.
+
+
+## Create the Service File
+
+Create `/etc/systemd/system/monerod.service` with the following content:
+
+    [Unit]
+    Description=Monero Daemon
+    After=network.target
+
+    [Service]
+    User=monero
+    Group=monero
+    WorkingDirectory=~
+
+    ExecStart=/your/path/to/monerod --non-interactive --config-file /etc/monerod.conf
+
+    Restart=always
+    PrivateTmp=true
+
+    [Install]
+    WantedBy=multi-user.target
```

Comment:
This would be better off by linking to whichever config file and systemd file is in-tree. When the in-tree files are updated, the site won't require updating.

Reply:
Sounds good, I'll add a link & replace the content with a wget commands to the files on github.

Reply:
Why not link to https://github.com/monero-project/monero/blob/master/utils/systemd/monerod.service?